docs/Cray-support.rst

```diff
@@ -0,0 +1,162 @@
+.. _cray_support:
+
+EasyBuild on Cray
+=================
+
+As of EasyBuild v2.7.0, the support for using EasyBuild on Cray systems is considered stable.
+Tt enables building/installing software using the ``PrgEnv`` modules provided by Cray.
+This page provides an overview of the current status.
+
+For more information about this, contact:
+
+* Guilherme Peretti-Pezzi (CSCS, Switserland, peretti@cscs.ch)
+* Kenneth Hoste (HPC-UGent, Belgium; kenneth.hoste@ugent.be)
+* Petar Forai (IMBA/IMP, Austria; petar.forai@imp.ac.at).
+
+Thanks to:
+
+* Olli-Pekka Letho (CSC.fi)
+* Tim Robinson and Guilherme Peretti-Pezzi (CSCS.ch)
+* Eric Bavier (Cray)
+* Brett Bode (NCSA)
+
+for providing us access to Cray systems, for their support and for testing and contributing to this work.
+
+Test systems
+------------
+
+* Piz Daint & Piz Dora @ CSCS.ch (http://www.cscs.ch/computers/piz_daint_piz_dora/index.html)
+* Santis & Brisi (TDS) @ CSCS.ch
+* Sisu @ CSC.fi (https://research.csc.fi/sisu-supercomputer)
+* Swan (TDS) @ Cray
+* Blue Waters @ NCSA
+
+EasyBuild toolchains
+--------------------
+
+* ``CrayCCE``: ``PrgEnv-cray`` with pinned versions of ``cce``, ``cray-libsci`` and ``cray-mpich``
+* ``CrayGNU``: ``PrgEnv-gnu`` with pinned versions of ``gcc``, ``cray-libsci`` and ``cray-mpich``
+* ``CrayIntel``: ``PrgEnv-intel`` with pinned versions of ``intel``, ``cray-libsci`` and ``cray-mpich``
+* ``CrayPGI``: ``PrgEnv-pgi`` with pinned versions of ``pgi`` and ``cray-mpich``
+
+versions:
+
+* ``Cray{CCE,GNU,Intel}/2015.06`` (requires Cray PE June/2015)
+* ``Cray{CCE,GNU,Intel}/2015.11`` (requires Cray PE November/2015)
+* ``CrayGNU/2016.03`` (requires Cray PE March/2016)
+* ``Cray{GNU,PGI}/2016.04`` (requires Cray PE April/2016)
+* ``Cray{GNU,Intel}/2016.06`` (requires Cray PE June/2016)
+
+What works already?
+-------------------
+(see below for more information)
+
+*  **HPL** (LINPACK) benchmark version 2.1
+
+Major scientific software applications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* **CP2K** 2.6.0
+* **GROMACS** 4.6.7
+* **Python** 2.7.9 + numpy 1.9.2 + scipy 0.15.1
+* **WRF** 3.6.1 (pending on Sisu)
+
```

Comment:
How about adding a link to the public [CSCS production repository](https://github.com/eth-cscs/production/) somewhere? There we have an updated list of [supported applications in Piz Daint](https://github.com/eth-cscs/production/blob/master/jenkins-builds/6.0.UP02-2016.11-gpu)